Vision/samples/V1/SamplegenBasics.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * DO NOT EDIT! This is a generated sample ("Request",  "samplegen_basics")
+ */
+
+// sample-metadata
+//   title: This is the sample title
+//   description: This is the sample description
+//   usage: php samples/V1/SamplegenBasics.php [--display_name "This is the default value of the display_name request field"]
+// [START samplegen_basics]
+require __DIR__.'/../../vendor/autoload.php';
```

Comment:
ℹ️ **Known Issue:** we will remove `require` from the [START/END] snippet. Snippet will begin with `use` statements.